Jenkinsfile

```diff
@@ -118,6 +118,7 @@ def runRelease(architectures) {
             }
         }
 
+        // Download cert files, to be used for signing the Windows executable, packaged by Chocolatey.
```

Comment:
We sign the binary also for the standalone Windows executable, and not just for Windows executable packaged insithe Chocolaty.